payment-service/src/main/java/com/hedvig/paymentservice/services/segmentPublisher/TrustlyEventListener.kt

```diff
@@ -21,7 +17,7 @@ import org.springframework.stereotype.Component
 @Profile("customer.io")
 @ProcessingGroup("SegmentProcessorGroup")
```

Comment:
Will it mess up something if we change this to 
```suggestion
@ProcessingGroup("TrustlySegmentProcessorGroup")
```
I don't know axon to well

Reply:
Yea that names makes way more sense but I think if you change it it will lose track of where it was in the event tracking token